### ISSUE TITLE
Can no longer Sync Sketch files to Avocode

### ISSUE THREAD

**Issue by gukelbe** [avocode/avocode#638]
IMPORTANT: Please try to describe how did it happen.

Updated to OSX 10.10 yesterday because I could not update to the latest Sketch until I did. Updated Sketch to 3.5.1. Did not sync at all yesterday. Today updated Avocode to 2.5.2 and Sketch Avocode plugin to 2.1.5 tried to sync. I have developers waiting on me and now I get this error:
-----
Sketch Exporter 2.1.5 running in Sketch 3.5.1
```
Error: Error: Command failed: 2016-01-27 12:26:57.936 sketchtool[2072:23735] Error: caught exception whilst suppressing update events: *** -[MSUnarchiver decodeObjectForKey:]: cannot decode object of class (MSImmutableSharedStyleContainer) for key (layerStyles)

**Comment by gukelbe** [comment]
Also did a test. Created a new Sketch file with nothing in it. Tried to sync as a new project and go the same error

**Comment by bolekkerous** [comment]
Hello!
We have already released Sketch Extension 3.0.0-prerelease4 with Sketch 3.5 support. Please download the latest version from https://github.com/avocode/avocode/blob/master/sketch-extension-beta.md#installation--update

I am closing this issue as a duplicate, if there would be a problem with the new extension, please follow up on #609

Thank you and sorry for the inconvenience.
